gitflow_toolbox/check_branch_exists.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+import click
+
+from gitflow_toolbox.common.gitlab import CurrentGitlab, RemoteGitlab
+from gitflow_toolbox.common.is_main_call import is_main_call
+
+
+@click.command()
+@click.option("--remote/--local", default=False)
+@click.argument("branch", type=str)
+@click.pass_context
+def check_branch_exists(ctx, remote, branch):
+    """Checks if a branch exists
+
+    Args:
+        remote (boolean): whether to check on the current gitlab or remote gitlab (True=remote)
+        branch (str): branch name
+    """
+    click.echo(f"Checking if {branch} branch exists...")
+    project = RemoteGitlab().remote_project() if remote else CurrentGitlab().current_project()
+    branch_exists = branch in [b.name for b in project.branches.list()]
```

Comment:
Hum, a better way to do it is by using next => use iterator and exit on the first match, prevent a loop on a computed loop.

```suggestion
    branch_exists = next((True for b in project.branches.list() if b.name == branch), False)
```

Reply:
Nice one !